src/packages/sdk/src/cli/getSchema.ts

```diff
@@ -159,38 +220,88 @@ export async function getSchema(schemaPathFromArgs?): Promise<string> {
  * Sync
  */
 
-export function getSchemaPathSync(schemaPathFromArgs?): string | null {
+export function getSchemaPathSync(schemaPathFromArgs?: string) {
+  return getSchemaPathSyncInternal(schemaPathFromArgs, {
+    cwd: process.cwd(),
+  })
+}
+
+export function getSchemaPathSyncInternal(
+  schemaPathFromArgs?: string,
+  opts: { cwd: string } = {
+    cwd: process.cwd(),
+  },
+): string | null {
```

Comment:
Not sure whether we actually want to make this signature the exported one on the SDK. While it was made to ease testing, it could be useful for SDK tinkerers. Same goes for the `getSchemaPathInternal`.

Reply:
(not clear on the highlighted code, but the exported `getSchemaPathSync` function now just calls that "internal" one)